src/components/LoginForm/index.tsx

```diff
@@ -1,17 +1,107 @@
 import Button from '../styled-components/Button';
+import { useState, useCallback } from 'react';
+import { Link } from 'react-router-dom';
 import './style.scss';
 
 const LoginForm = () => {
+  const PW_REQUIRE_CHECK = '비밀번호를 입력해주세요.';
+  const PW_VALID_CHECK = '8자 이하로 입력해주세요.';
+
+  // login ID, PW
+  const [userId, setUserId] = useState<string>('');
+  const [userPw, setUserPw] = useState<string>('');
+  const [idValid, setIdValid] = useState<boolean>(false);
+  const [pwValid, setPwValid] = useState<boolean>(false);
+  const [pwdMsg, setPwdMsg] = useState<string>('');
+
+  // Id focus
+  const userIdInput = useCallback((inputElement) => {
+    if (inputElement) {
+      inputElement.focus();
+    }
+  }, []);
+
+  // ID input Change Event
+  const onChangeIdInput = (e) => {
+    setUserId(e.target.value);
+    setIdValid(false);
+  };
+
+  // PW input Change Event
+  const onChangePwInput = (e) => {
+    setUserPw(e.target.value);
+    setPwValid(false);
+  };
+
   const onClickEvent = () => {
-    // eslint-disable-next-line no-console
+    // 회원가입 작업 이후 axios 구현 예정
     console.log('You clicked !');
   };
+
+  // userID focus out
+  const onBlurIdInput = (e) => {
+    if (!e.currentTarget.value) {
+      setIdValid(true);
+    }
+  };
+
+  // userID focus out
+  const onBlurPwInput = (e) => {
```

Comment:
```suggestion
  const onBlurPwInput = (e: SyntheticEvent<HTMLInputElement>) => {
```

event 타입도 정해주면 어떨까요?